src/Cms/ModelPermissions.php

```diff
@@ -95,12 +97,14 @@ public function can(string $action): bool
 			}
 		}
 
-		return $this->permissions->for($this->category, $action);
+		return $this->permissions->for($this->category, $action, $default);
 	}
 
-	public function cannot(string $action): bool
-	{
-		return $this->can($action) === false;
+	public function cannot(
+		string $action,
+		bool $default = false
```

Comment:
This is creating a knot in my brain. I understand why you chose false here, but is it still logical? Wouldn't you read it like true === cannot & false === can?

Reply:
I see your point. I think we could do for the parameter `bool $default = true` and then the code reads something like this?

```php
return $this->can($action, !$default) === false;
```

Reply:
Yes, I really just wonder which way is better. It's one of those situations where both solutions are correct and logically explainable and no matter how we solve it, it will always be one of those arguments where you need to look up how it was meant to be used afterwards.

Reply:
I would say `bool $default = true` and `$this->can($action, !$default) === false` makes most sense. And then a DocBlock comment:

```
@param bool $default Will be returned if $action does not exist
```

Reply:
Changed it.